plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -183,28 +226,40 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 			if match := regExpMatch(namespaceStrPattern, dstNamespace); !match {
 				log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
 					"failed to match namespace %s with K8s Namespace pattern %s", dstNamespace, namespaceStrPattern)
+				alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Invalid destination namespace string, "+
```

Comment:
Is there any naming requirement for service just like Pod?

Reply:
Updated previous validation code with the validation functions provided by K8S like [this](https://github.com/kubernetes/kubernetes/blob/b46612a74224b0871a97dae819f5fb3a1763d0b9/pkg/apis/core/validation/validation.go#L215).

Reply:
Also added the validation for service name.